mkdocs.yml

```diff
@@ -280,7 +280,7 @@ nav:
 #        - Format.xml: developer-guide/etendo-classic/concepts/Format_xml.md
         - Inventory Status: developer-guide/etendo-classic/concepts/inventory-status.md
 #        - Java Coding Conventions: developer-guide/etendo-classic/concepts/Java_Coding_Conventions.md
-#        - JavaScript Coding Conventions: developer-guide/etendo-classic/concepts/JavaScript_Coding_Conventions.md
+#        - JavaScript Coding Conventions: developer-guide/etendo-classic/concepts/javascript-coding-conventions.md
```

Comment:
@Tomas-Vesco Rename the file developer-guide/etendo-classic/concepts/javascript-coding-conventions.md as well.